src/components/biggive-formatted-text/biggive-formatted-text.tsx

```diff
@@ -21,6 +21,9 @@ export class BiggiveFormattedText {
   @Prop() maxWidth: number = 100;
 
   render() {
-    return <div class={'container max-width-' + this.maxWidth + ' text-colour-' + this.defaultTextColour + ' space-below-' + this.spaceBelow}></div>;
+    return;
+    <div class={'container max-width-' + this.maxWidth + ' text-colour-' + this.defaultTextColour + ' space-below-' + this.spaceBelow}>
+      <slot></slot>
+    </div>;
```

Comment:
this is unreachable code after the return statement - not sure what you're trying to do here.